### ISSUE TITLE
Install mynode on a running LVM root

### ISSUE THREAD

**Issue by as-schug** [mynodebtc/mynode#700]
Installing mynode on a running LVM root filesystem does not start

It would be great if we can run that on a lvm or even a raid root partition.

This is the result from /usr/bin/mynode_startup.sh script
++ WEBSSH2_VERSION=v0.2.10-0
++ WEBSSH2_VERSION_FILE=/mnt/hdd/mynode/settings/webssh2_version
++ WEBSSH2_LATEST_VERSION_FILE=/mnt/hdd/mynode/settings/webssh2_version_latest
++ PYTHON_VERSION=3.8.9
++ PYTHON_ARM32_GRPCIO_VERSION=1.40.0
++ NODE_JS_VERSION=14.x
++ NODE_NPM_VERSION='^8.1.0'
++ RUST_VERSION=1.56.1
++ '[' -f /usr/share/mynode/mynode_app_versions_custom.sh ']'
++ '[' -f /mnt/hdd/mynode/settings/mynode_app_versions_custom.sh ']'
+ '[' '!' -w / ']'
+ chmod +t /tmp
+ settings_file_exists skip_backup_dns_servers
+ '[' -f /home/bitcoin/.mynode/skip_backup_dns_servers ']'
+ '[' -f /mnt/hdd/mynode/settings/skip_backup_dns_servers ']'
+ return 1
+ needDns=0
+ grep 'Added at myNode startup' /etc/resolv.conf
# Added at myNode startup
+ '[' 0 = 1 ']'
+ sed -i 's/^.*append domain-name-servers .*/append domain-name-servers 1.1.1.1, 208.67.222.222, 8.8.8.8;/g' /etc/dhcp/dhclient.conf
+ '[' -d /sys/bus/usb/devices/ ']'
+ for dev in /sys/bus/usb/devices/*/power/control
+ echo on
+ for dev in /sys/bus/usb/devices/*/power/control
+ echo on
+ mkdir -p /home/admin/.config/
+ chown -R admin:admin /home/admin/.config/
+ mkdir -p /var/lib/mynode
+ '[' '!' -f /var/lib/mynode/.expanded_rootfs ']'
+ '[' 0 -eq 1 ']'
+ '[' 0 = 1 ']'
+ '[' 1 = 1 ']'
++ cut -d ' ' -f1
++ grep ' / '
++ mount
+ X86_ROOT_PARTITION=/dev/mapper/mynodevm--vg-root
++ lsblk -no pkname /dev/mapper/mynodevm--vg-root
+ X86_DEVICE=
+ X86_DEVICE_PATH=/dev/
++ grep -c '[0-9]'
++ cat /proc/partitions
+ X86_PARTITION_NUMBER=10
++ fdisk -l /dev/
fdisk: cannot open /dev/: Is a directory
++ grep Disklabel
+ X86_FDISK_TYPE=

**Comment by as-schug** [comment]
Looking closer to startup script, I can touch /var/lib/mynode/.expanded_rootfs, so script can run.
Is it safe to touch /var/lib/mynode/.expanded_rootfs and let mynode run in production?

**Comment by tehelsper** [comment]
Yes, touching that file is totally fine if you don't want it to try to expand the rootfs. I have not tried on an LVM setup - I imagine everything should work except for that step.

I'll have to try it one day and get the script to automatically skip that step.

**Comment by as-schug** [comment]
> Yes, touching that file is totally fine if you don't want it to try to expand the rootfs. I have not tried on an LVM setup - I imagine everything should work except for that step.
> 
> I'll have to try it one day and get the script to automatically skip that step.

All is working fine since that touch...
